### PR TITLE
[doc](stream load) Links occasionally fail when page jumps

### PR DESCRIPTION
**Pull request by wanghuan2054**
Links occasionally fail when page jumps , so use http access paths instead of relative paths

### CODE REVIEW

**Comment by luzhijing** [comment]
@KassieZ

**Review by luzhijing — APPROVED** [review]
LGTM

**Review by luzhijing — APPROVED** [review]
LGTM

**Review by shasevendow — APPROVED** [review]

File: docs/data-operate/import/stream-load-manual.md
Comment (KassieZ) [inline docs/data-operate/import/stream-load-manual.md:1]: Hihi , please use relative path instead of absolute path.

Comment (wanghuan2054) [reply docs/data-operate/import/stream-load-manual.md:1]: OK

**Pull request closed without merging**